src/main/app/case/answers/getAnswerRows.ts

```diff
@@ -23,7 +27,12 @@ export const getAnswerRows = function (section: Sections): GovUkNunjucksSummary[
     formState: Partial<Case>;
   } = this.ctx;
 
-  const stepsWithContent = isApplicant2 ? stepsWithContentApplicant2 : stepsWithContentApplicant1;
+  let stepsWithContent = isApplicant2 ? stepsWithContentApplicant2 : stepsWithContentApplicant1;
+  if (overrideStepsContent === 1) {
+    stepsWithContent = stepsWithContentApplicant1;
```

Comment:
Do you need to assign `stepsWithContentApplicant1` to `stepsWithContent` again here? What does overrideStepsContent do that the isApplicant2 check doesn't?

Reply:
As they are app2, theyll only get app2 content displaying but check joint page wants app1 answers, hence i thought of this override variable.